### PR TITLE
Newsroom header and footer design

### PR DESCRIPTION
**Pull request by kaylaziegler**
Story: https://trello.com/c/yjIJd3EP

See [Trello comment](https://trello.com/c/yjIJd3EP/680-as-a-newsroom-user-when-i-visit-the-homepage-i-want-to-see-the-basic-page-layout-with-header-and-footer#comment-5a500013a3af4e6153d68903) for notes about what is and isn't included in this PR.

### CODE REVIEW

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -0,0 +1,80 @@
+/*
```
Comment (tatwell) [inline app/assets/stylesheets/application.css.scss]: Is `.css.scss` necessary or can we simply use `.scss` extension? If so, I'd prefer that.

File: app/assets/stylesheets/header.css.scss [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+  @include breakpoint($bp-tablet) {
+    padding-top: 120px;
+  }
+}
+
+#omni-nav-v2 {
+  position: fixed;
+  width: 100%;
+  top: 0;
+  font-family: "futura-pt", verdana, sans-serif;
```
Comment (tatwell) [inline app/assets/stylesheets/header.css.scss]: This looks the same as `$default-fonts`.

File: app/assets/stylesheets/header.css.scss [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+            }
+
+            .login-menu {
+              visibility: hidden;
+              opacity: 0;
+              position: absolute;
+              top: $primary-nav-height;
+              right: -20px;
+              width: 175px;
+              background-color: #2f2f2f;
```
Comment (tatwell) [inline app/assets/stylesheets/header.css.scss]: This looks like `$chapman-black`. Double-check settings and try to use vars where possible.

File: app/assets/stylesheets/variables.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// Breakpoints
```
Comment (tatwell) [inline app/assets/stylesheets/variables.scss]: Should all variables be in this file?

File: app/assets/stylesheets/header.css.scss (hunk 3 above)
Comment (kaylaziegler) [reply app/assets/stylesheets/header.css.scss]: Sorry I should have double checked them all at the end. The redundancies have been removed.

File: app/assets/stylesheets/header.css.scss (hunk 2 above)
Comment (kaylaziegler) [reply app/assets/stylesheets/header.css.scss]: It isn't, but I did see some places where I could have been using the `$default-fonts` and fixed it.

**Pull request merged**